.github/workflows/lint-participant-manager-datastore.yml

```diff
@@ -0,0 +1,27 @@
+name: lint-participant-manager-datastore
```

Comment:
Should this have the copyright header?

Reply:
YAML does not support multi line comments.